### PR TITLE
issue_1277: Fixed finder lib errors handling

### PR DESCRIPTION
**Pull request by arslanbekova**

### CODE REVIEW

File: src/pageServices/contentScripts/generationData.js [hunk 1]
```diff
@@ -66,18 +66,32 @@ export const getGenerationAttributes = () => {
...
+    // '#Country_16856319000360.6264611129794591' is not a valid selector."
+    // And because of lib logic we can't prevent generation these selectors.
+    // Reproduced on https://www.docker.com
+    try {
+      const finderForbiddenAttributes = ["jdn-hash", "href", "class", "xmlns", "xmlns:xlink", "xlink:href"];
+      const selectorByFinder = await finder(element, {
+        attr: (name, value) => {
+          console.log(name);
+          console.log(value);
+          return value && !value.includes(".") && !finderForbiddenAttributes.includes(name);
```
Comment (MariiaNebesnova) [inline src/pageServices/contentScripts/generationData.js:87]: if `!value.includes(".")` doesn't work, no need to keep this code

File: src/pageServices/contentScripts/generationData.js [hunk 2]
```diff
@@ -66,18 +66,32 @@ export const getGenerationAttributes = () => {
...
+    // We get "Uncaught DOMException: Failed to execute 'querySelectorAll' on 'Document':
+    // '#Country_16856319000360.6264611129794591' is not a valid selector."
+    // And because of lib logic we can't prevent generation these selectors.
+    // Reproduced on https://www.docker.com
+    try {
+      const finderForbiddenAttributes = ["jdn-hash", "href", "class", "xmlns", "xmlns:xlink", "xlink:href"];
+      const selectorByFinder = await finder(element, {
+        attr: (name, value) => {
+          console.log(name);
+          console.log(value);
```
Comment (MariiaNebesnova) [inline src/pageServices/contentScripts/generationData.js:86]: clean up them, please

Comment (arslanbekova) [reply src/pageServices/contentScripts/generationData.js]: done

File: src/pageServices/contentScripts/generationData.js (hunk 1 above)
Comment (arslanbekova) [reply src/pageServices/contentScripts/generationData.js]: removed

**Pull request merged**